package.json

```diff
@@ -34,7 +34,9 @@
     "tsconfig-paths": "^4.1.0",
     "typescript": "^4.8.3"
   },
-  "dependencies": {},
+  "dependencies": {
+    "@solana/wallet-adapter-phantom": "^0.9.23"
```

Comment:
This is for VersionedTransaction addition to solana web3 library. phantom adapter includes the VersionedTransaction